packages/magento-product/components/ProductSidebarDelivery/ProductSidebarDelivery.tsx

```diff
@@ -49,9 +50,11 @@ export function ProductSidebarDelivery(props: ProductSidebarDeliveryProps) {
       })}
     >
       <IconSvg src={iconOrderBefore} size='small' sx={{ gridArea: 'image' }} />
-      <Box sx={{ typography: 'body2', gridArea: 'title', fontWeight: 600 }}>{title}</Box>
+      <Box sx={{ typography: 'body2', gridArea: 'title', fontWeight: 600 }}>
+        <Trans id={title} />
+      </Box>
       <Box sx={{ typography: 'body2', gridArea: 'subtitle', color: 'text.primary' }}>
-        {subtitle}
+        <Trans id={subtitle} />
```

Comment:
This isn't allowed, this breaks the static analysis (extraction of possible strings). Add the Trans on the usage side.